addons/test_event_full/tests/test_event_mail.py

```diff
@@ -327,3 +327,17 @@ def test_event_mail_on_sale_confirmation(self):
                 "email_from": self.test_event.organizer_id.email_formatted,
             },
         )
+
+    def test_template_translation(self):
```

Comment:
Would be great to use a test user using ``@users`` decorator :) just the template update will probably require to be done as sudo but that way we test using real ACLs